### ISSUE TITLE
npm binary dependencies

### ISSUE THREAD

**Issue by one-pereyra** [fractal-code/meteor-azure#20]
I am having issues with npm binary dependencies;

> D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\npm.cmd
> rebuild --update-binary
> node-pre-gyp ERR! Tried to download:
> https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.0.14/fse-v1.0.14-node-v46-win32-ia32.tar.gz 
> node-pre-gyp ERR! Pre-built binaries not found for fsevents@1.0.14 and
> node@4.6.2 (node-v46 ABI) (falling back to source compile with node-gyp) 
> 

Try with `windows-build-tools` but got access error;

> Please restart this script from an administrative PowerShell!
> The build tools cannot be installed without administrative rights.
> To fix, right-click on PowerShell and run "as Administrator".
> 

Did you come across a similar issue? Do you have any ideas?

**Comment by ramijarrar** [comment]
Do you have the full deployment log? What version of the script are you using?

**Comment by one-pereyra** [comment]
I am using 1.4.5.
see attached
[deployment_log.txt](https://github.com/talos-code/meteor-azure/files/770199/deployment_log.txt)

 deployment log.

**Comment by ramijarrar** [comment]
Can you paste with formatting?

**Comment by one-pereyra** [comment]
this is the best I got from azure;

```
Command: bash .config/azure/deploy.sh
meteor-azure: Setting Node version
Downloading node.js version 4.6.2 (32-bit)... Complete
Creating D:/home/meteor-azure/nvm\temp

Downloading npm version 2.15.11... Complete
Installing npm v2.15.11...

Installation complete. If you want to use this version, type

nvm use 4.6.2
meteor-azure: Now using Node v4.6.2 (32-bit)
meteor-azure: Setting NPM version
D:\home\meteor-azure\nvm\v4.6.2\npm -> D:\home\meteor-azure\nvm\v4.6.2\node_modules\npm\bin\npm-cli.js
npm@3.10.9 D:\home\meteor-azure\nvm\v4.6.2\node_modules\npm
meteor-azure: Now using NPM v3.10.9
meteor-azure: Installing JSON tool
D:\home\meteor-azure\nvm\v4.6.2\json -> D:\home\meteor-azure\nvm\v4.6.2\node_modules\json\lib\json.js
D:\home\meteor-azure\nvm\v4.6.2
`-- json@9.0.4 

meteor-azure: Installing NPM dependencies

> windows-build-tools@1.1.0 postinstall D:\home\meteor-azure\nvm\v4.6.2\node_modules\windows-build-tools
> node ./lib/index.js

Downloading BuildTools_Full.exe
[>                                            ] 0.0% (0 B/s)
�[1000D�[0K�[1A�[1000D�[0K�[1A�[1000D�[0KDownloading BuildTools_Full.exe
[>                                            ] 0.0% (0 B/s)
Downloaded BuildTools_Full.exe. Saved to D:\local\UserProfile\.windows-build-tools\BuildTools_Full.exe.
�[1000D�[0K�[1A�[1000D�[0K�[1A�[1000D�[0KDownloading python-2.7.11.msi
[>                                            ] 0.0% (0 B/s)
�[1000D�[0K�[1A�[1000D�[0K�[1A�[1000D�[0KDownloading python-2.7.11.msi
[>                                            ] 0.0% (0 B/s)
Downloaded python-2.7.11.msi. Saved to D:\local\UserProfile\.windows-build-tools\python-2.7.11.msi.
Starting installation...
Please restart this script from an administrative PowerShell!
The build tools cannot be installed without administrative rights.
To fix, right-click on PowerShell and run "as Administrator".
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "D:\\home\\meteor-azure\\nvm\\v4.6.2\\node.exe" "D:\\home\\meteor-azure\\nvm\\v4.6.2\\node_modules\\npm\\bin\\npm-cli.js" "install" "--global" "--production" "windows-build-tools"
npm ERR! node v4.6.2
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE

npm ERR! windows-build-tools@1.1.0 postinstall: `node ./lib/index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the windows-build-tools@1.1.0 postinstall script 'node ./lib/index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the windows-build-tools package,
meteor-azure: Building app %DEPLOYMENT_TEMP%
npm ERR! not with npm itself.
meteor-azure: Building app + %DEPLOYMENT_TEMP%
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./lib/index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs windows-build-tools
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls windows-build-tools
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\home\site\repository\npm-debug.log
pnyx:boot: updating npm dependencies -- cryptico-js, deasync...
XMLTemplates: updating npm dependencies -- handlebars...
vazco:xml-template: updating npm dependencies -- handlebars...
pnyx:soap: updating npm dependencies -- xml4js, soap-vfork, jstoxml,
xml2js...
=> Errors while initializing project:

While loading package momentjs:moment@2.15.2:
error: Command failed:
D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\npm.cmd
rebuild --update-binary
node-pre-gyp ERR! Tried to download:
https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.0.14/fse-v1.0.14-node-v46-win32-ia32.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for fsevents@1.0.14 and
node@4.6.2 (node-v46 ABI) (falling back to source compile with node-gyp) 
Traceback (most recent call last):
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\gyp_main.py",
line 16, in <module>
sys.exit(gyp.script_main())
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",
line 545, in script_main
return main(sys.argv[1:])
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",
line 538, in main
return gyp_main(args)
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",
line 523, in gyp_main
generator.GenerateOutput(flat_list, targets, data, params)
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",
line 2003, in GenerateOutput
sln_projects, project_objects, flat=msvs_version.FlatSolution())
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",
line 1762, in _GatherSolutionFolders
return _DictsToFolders('', root, flat)
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",
line 1715, in _DictsToFolders
for folder, contents in bucket.iteritems():
AttributeError: 'MSVSProject' object has no attribute 'iteritems'
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit
(D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\lib\configure.js:305:16)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit
(internal/child_process.js:211:12)
gyp ERR! System Windows_NT 6.2.9200
gyp ERR! command
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\node-gyp\\bin\\node-gyp.js"
"configure" "--fallback-to-build"
"--module=D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\lib\\binding\\Release\\node-v46-win32-ia32\\fse.node"
"--module_name=fse"
"--module_path=D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\lib\\binding\\Release\\node-v46-win32-ia32"
gyp ERR! cwd
D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents
gyp ERR! node -v v4.6.2
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute
'D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\node.exe
D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\bin\node-gyp.js
configure --fallback-to-build
--module=D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\lib\binding\Release\node-v46-win32-ia32\fse.node
--module_name=fse
--module_path=D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\lib\binding\Release\node-v46-win32-ia32'
(1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous>
(D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\node_modules\node-pre-gyp\lib\util\compile.js:83:29)
node-pre-gyp ERR! stack     at emitTwo (events.js:87:13)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
node-pre-gyp ERR! stack     at maybeClose
(internal/child_process.js:854:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit
(internal/child_process.js:222:5)
node-pre-gyp ERR! System Windows_NT 6.2.9200
node-pre-gyp ERR! command
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
"D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\node_modules\\node-pre-gyp\\bin\\node-pre-gyp"
"install" "--fallback-to-build"
node-pre-gyp ERR! cwd
D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents
node-pre-gyp ERR! node -v v4.6.2
node-pre-gyp ERR! node-pre-gyp -v v0.6.29
node-pre-gyp ERR! not ok 

npm ERR! Windows_NT 6.2.9200
npm ERR! argv
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"
"rebuild" "--update-binary"
npm ERR! node v4.6.2
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! fsevents@1.0.14 install: `node-pre-gyp install
--fallback-to-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fsevents@1.0.14 install script 'node-pre-gyp
install --fallback-to-build'.
npm ERR! Make sure you have the latest version of node.js and npm
installed.
npm ERR! If you do, this is most likely a problem with the fsevents
package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get information on how to open an issue for this project
with:
npm ERR!     npm bugs fsevents
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls fsevents
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!
D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\npm-debug.log
node-pre-gyp ERR! Tried to download:
https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.0.14/fse-v1.0.14-node-v46-win32-ia32.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for fsevents@1.0.14 and
node@4.6.2 (node-v46 ABI) (falling back to source compile with node-gyp) 
Traceback (most recent call last):
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\gyp_main.py",
line 16, in <module>
sys.exit(gyp.script_main())
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",
line 545, in script_main
return main(sys.argv[1:])
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",
line 538, in main
return gyp_main(args)
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",
line 523, in gyp_main
generator.GenerateOutput(flat_list, targets, data, params)
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",
line 2003, in GenerateOutput
sln_projects, project_objects, flat=msvs_version.FlatSolution())
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",
line 1762, in _GatherSolutionFolders
return _DictsToFolders('', root, flat)
File
"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",
line 1715, in _DictsToFolders
for folder, contents in bucket.iteritems():
AttributeError: 'MSVSProject' object has no attribute 'iteritems'
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit
(D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\lib\configure.js:305:16)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit
(internal/child_process.js:211:12)
gyp ERR! System Windows_NT 6.2.9200
gyp ERR! command
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\node-gyp\\bin\\node-gyp.js"
"configure" "--fallback-to-build"
"--module=D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\lib\\binding\\Release\\node-v46-win32-ia32\\fse.node"
"--module_name=fse"
"--module_path=D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\lib\\binding\\Release\\node-v46-win32-ia32"
gyp ERR! cwd
D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents
gyp ERR! node -v v4.6.2
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute
'D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\node.exe
D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\bin\node-gyp.js
configure --fallback-to-build
--module=D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\lib\binding\Release\node-v46-win32-ia32\fse.node
--module_name=fse
--module_path=D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\lib\binding\Release\node-v46-win32-ia32'
(1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous>
(D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\node_modules\node-pre-gyp\lib\util\compile.js:83:29)
node-pre-gyp ERR! stack     at emitTwo (events.js:87:13)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
node-pre-gyp ERR! stack     at maybeClose
(internal/child_process.js:854:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit
(internal/child_process.js:222:5)
node-pre-gyp ERR! System Windows_NT 6.2.9200
node-pre-gyp ERR! command
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
"D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\node_modules\\node-pre-gyp\\bin\\node-pre-gyp"
"install" "--fallback-to-build"
node-pre-gyp ERR! cwd
D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents
node-pre-gyp ERR! node -v v4.6.2
node-pre-gyp ERR! node-pre-gyp -v v0.6.29
node-pre-gyp ERR! not ok 

npm ERR! Windows_NT 6.2.9200
npm ERR! argv
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"
"rebuild" "--update-binary"
npm ERR! node v4.6.2
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! fsevents@1.0.14 install: `node-pre-gyp install
--fallback-to-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fsevents@1.0.14 install script 'node-pre-gyp
install --fallback-to-build'.
npm ERR! Make sure you have the latest version of node.js and npm
installed.
npm ERR! If you do, this is most likely a problem with the fsevents
package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get information on how to open an issue for this project
with:
npm ERR!     npm bugs fsevents
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls fsevents
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!
D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\npm-debug.log


While building package pnyx:soap:
error: couldn't install npm package xml4js@0.1.3: Command failed:
D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\npm.cmd
install xml4js@0.1.3
npm ERR! addLocal Could not install
D:\local\Temp\npm-11544-dd92a154\git-cache-321e68d0\bb7889f02a774f347b184f3f97e8b7d8cde15e7a
npm ERR! Windows_NT 6.2.9200
npm ERR! argv
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"
"install" "xml4js@0.1.3"
npm ERR! node v4.6.2
npm ERR! npm  v3.10.9
npm ERR! path
D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz
npm ERR! code ENOTSUP
npm ERR! errno -4049
npm ERR! syscall scandir

npm ERR! nospc ENOTSUP: operation not supported on socket, scandir
'D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz'
npm ERR! nospc This is most likely not a problem with npm itself
npm ERR! nospc and is related to insufficient space on your system.

npm ERR! Please include the following file with any support request:
npm ERR!
D:\home\site\repository\packages\pnyx-soap\.npm\package-new-oh33ut\npm-debug.log
npm ERR! addLocal Could not install
D:\local\Temp\npm-11544-dd92a154\git-cache-321e68d0\bb7889f02a774f347b184f3f97e8b7d8cde15e7a
npm ERR! Windows_NT 6.2.9200
npm ERR! argv
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"
"install" "xml4js@0.1.3"
npm ERR! node v4.6.2
npm ERR! npm  v3.10.9
npm ERR! path
D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz
npm ERR! code ENOTSUP
npm ERR! errno -4049
npm ERR! syscall scandir

npm ERR! nospc ENOTSUP: operation not supported on socket, scandir
'D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz'
npm ERR! nospc This is most likely not a problem with npm itself
npm ERR! nospc and is related to insufficient space on your system.

npm ERR! Please include the following file with any support request:
npm ERR!
D:\home\site\repository\packages\pnyx-soap\.npm\package-new-oh33ut\npm-debug.log


meteor-azure: ERROR! Could not generate Meteor bundle
npm ERR! Windows_NT 6.2.9200\r\nnpm ERR! argv "D:\\home\\meteor-azure\\nvm\\v4.6.2\\node.exe" "D:\\home\\meteor-azure\\nvm\\v4.6.2\\node_modules\\npm\\bin\\npm-cli.js" "install" "--global" "--production" "windows-build-tools"\r\nnpm ERR! node v4.6.2\r\nnpm ERR! npm  v3.10.9\r\nnpm ERR! code ELIFECYCLE\r\n\r\nnpm ERR! windows-build-tools@1.1.0 postinstall: `node ./lib/index.js`\r\nnpm ERR! Exit status 1\r\nnpm ERR! \r\nnpm ERR! Failed at the windows-build-tools@1.1.0 postinstall script 'node ./lib/index.js'.\r\nnpm ERR! Make sure you have the latest version of node.js and npm installed.\r\nnpm ERR! If you do, this is most likely a problem with the windows-build-tools package,\r\nnpm ERR! not with npm itself.\r\nnpm ERR! Tell the author that this fails on your system:\r\nnpm ERR!     node ./lib/index.js\r\nnpm ERR! You can get information on how to open an issue for this project with:\r\nnpm ERR!     npm bugs windows-build-tools\r\nnpm ERR! Or if that isn't available, you can get their info via:\r\nnpm ERR!     npm owner ls windows-build-tools\r\nnpm ERR! There is likely additional logging output above.\r\n\r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!     D:\home\site\repository\npm-debug.log\r\n=> Errors while initializing project:\r\n\r\nWhile loading package momentjs:moment@2.15.2:\r\nerror: Command failed:\r\nD:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\npm.cmd\r\nrebuild --update-binary\r\nnode-pre-gyp ERR! Tried to download:\r\nhttps://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.0.14/fse-v1.0.14-node-v46-win32-ia32.tar.gz \r\nnode-pre-gyp ERR! Pre-built binaries not found for fsevents@1.0.14 and\r\nnode@4.6.2 (node-v46 ABI) (falling back to source compile with node-gyp) \r\nTraceback (most recent call last):\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\gyp_main.py",\r\nline 16, in <module>\r\nsys.exit(gyp.script_main())\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",\r\nline 545, in script_main\r\nreturn main(sys.argv[1:])\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",\r\nline 538, in main\r\nreturn gyp_main(args)\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",\r\nline 523, in gyp_main\r\ngenerator.GenerateOutput(flat_list, targets, data, params)\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",\r\nline 2003, in GenerateOutput\r\nsln_projects, project_objects, flat=msvs_version.FlatSolution())\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",\r\nline 1762, in _GatherSolutionFolders\r\nreturn _DictsToFolders('', root, flat)\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",\r\nline 1715, in _DictsToFolders\r\nfor folder, contents in bucket.iteritems():\r\nAttributeError: 'MSVSProject' object has no attribute 'iteritems'\r\ngyp ERR! configure error \r\ngyp ERR! stack Error: `gyp` failed with exit code: 1\r\ngyp ERR! stack     at ChildProcess.onCpExit\r\n(D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\lib\configure.js:305:16)\r\ngyp ERR! stack     at emitTwo (events.js:87:13)\r\ngyp ERR! stack     at ChildProcess.emit (events.js:172:7)\r\ngyp ERR! stack     at Process.ChildProcess._handle.onexit\r\n(internal/child_process.js:211:12)\r\ngyp ERR! System Windows_NT 6.2.9200\r\ngyp ERR! command\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\node-gyp\\bin\\node-gyp.js"\r\n"configure" "--fallback-to-build"\r\n"--module=D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\lib\\binding\\Release\\node-v46-win32-ia32\\fse.node"\r\n"--module_name=fse"\r\n"--module_path=D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\lib\\binding\\Release\\node-v46-win32-ia32"\r\ngyp ERR! cwd\r\nD:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\r\ngyp ERR! node -v v4.6.2\r\ngyp ERR! node-gyp -v v3.4.0\r\ngyp ERR! not ok \r\nnode-pre-gyp ERR! build error \r\nnode-pre-gyp ERR! stack Error: Failed to execute\r\n'D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\node.exe\r\nD:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\bin\node-gyp.js\r\nconfigure --fallback-to-build\r\n--module=D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\lib\binding\Release\node-v46-win32-ia32\fse.node\r\n--module_name=fse\r\n--module_path=D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\lib\binding\Release\node-v46-win32-ia32'\r\n(1)\r\nnode-pre-gyp ERR! stack     at ChildProcess.<anonymous>\r\n(D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\node_modules\node-pre-gyp\lib\util\compile.js:83:29)\r\nnode-pre-gyp ERR! stack     at emitTwo (events.js:87:13)\r\nnode-pre-gyp ERR! stack     at ChildProcess.emit (events.js:172:7)\r\nnode-pre-gyp ERR! stack     at maybeClose\r\n(internal/child_process.js:854:16)\r\nnode-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit\r\n(internal/child_process.js:222:5)\r\nnode-pre-gyp ERR! System Windows_NT 6.2.9200\r\nnode-pre-gyp ERR! command\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\node_modules\\node-pre-gyp\\bin\\node-pre-gyp"\r\n"install" "--fallback-to-build"\r\nnode-pre-gyp ERR! cwd\r\nD:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\r\nnode-pre-gyp ERR! node -v v4.6.2\r\nnode-pre-gyp ERR! node-pre-gyp -v v0.6.29\r\nnode-pre-gyp ERR! not ok \r\n\r\nnpm ERR! Windows_NT 6.2.9200\r\nnpm ERR! argv\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"\r\n"rebuild" "--update-binary"\r\nnpm ERR! node v4.6.2\r\nnpm ERR! npm  v3.10.9\r\nnpm ERR! code ELIFECYCLE\r\nnpm ERR! fsevents@1.0.14 install: `node-pre-gyp install\r\n--fallback-to-build`\r\nnpm ERR! Exit status 1\r\nnpm ERR! \r\nnpm ERR! Failed at the fsevents@1.0.14 install script 'node-pre-gyp\r\ninstall --fallback-to-build'.\r\nnpm ERR! Make sure you have the latest version of node.js and npm\r\ninstalled.\r\nnpm ERR! If you do, this is most likely a problem with the fsevents\r\npackage,\r\nnpm ERR! not with npm itself.\r\nnpm ERR! Tell the author that this fails on your system:\r\nnpm ERR!     node-pre-gyp install --fallback-to-build\r\nnpm ERR! You can get information on how to open an issue for this project\r\nwith:\r\nnpm ERR!     npm bugs fsevents\r\nnpm ERR! Or if that isn't available, you can get their info via:\r\nnpm ERR!     npm owner ls fsevents\r\nnpm ERR! There is likely additional logging output above.\r\n\r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!\r\nD:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\npm-debug.log\r\nnode-pre-gyp ERR! Tried to download:\r\nhttps://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.0.14/fse-v1.0.14-node-v46-win32-ia32.tar.gz \r\nnode-pre-gyp ERR! Pre-built binaries not found for fsevents@1.0.14 and\r\nnode@4.6.2 (node-v46 ABI) (falling back to source compile with node-gyp) \r\nTraceback (most recent call last):\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\gyp_main.py",\r\nline 16, in <module>\r\nsys.exit(gyp.script_main())\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",\r\nline 545, in script_main\r\nreturn main(sys.argv[1:])\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",\r\nline 538, in main\r\nreturn gyp_main(args)\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\__init__.py",\r\nline 523, in gyp_main\r\ngenerator.GenerateOutput(flat_list, targets, data, params)\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",\r\nline 2003, in GenerateOutput\r\nsln_projects, project_objects, flat=msvs_version.FlatSolution())\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",\r\nline 1762, in _GatherSolutionFolders\r\nreturn _DictsToFolders('', root, flat)\r\nFile\r\n"D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py",\r\nline 1715, in _DictsToFolders\r\nfor folder, contents in bucket.iteritems():\r\nAttributeError: 'MSVSProject' object has no attribute 'iteritems'\r\ngyp ERR! configure error \r\ngyp ERR! stack Error: `gyp` failed with exit code: 1\r\ngyp ERR! stack     at ChildProcess.onCpExit\r\n(D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\lib\configure.js:305:16)\r\ngyp ERR! stack     at emitTwo (events.js:87:13)\r\ngyp ERR! stack     at ChildProcess.emit (events.js:172:7)\r\ngyp ERR! stack     at Process.ChildProcess._handle.onexit\r\n(internal/child_process.js:211:12)\r\ngyp ERR! System Windows_NT 6.2.9200\r\ngyp ERR! command\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\node-gyp\\bin\\node-gyp.js"\r\n"configure" "--fallback-to-build"\r\n"--module=D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\lib\\binding\\Release\\node-v46-win32-ia32\\fse.node"\r\n"--module_name=fse"\r\n"--module_path=D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\lib\\binding\\Release\\node-v46-win32-ia32"\r\ngyp ERR! cwd\r\nD:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\r\ngyp ERR! node -v v4.6.2\r\ngyp ERR! node-gyp -v v3.4.0\r\ngyp ERR! not ok \r\nnode-pre-gyp ERR! build error \r\nnode-pre-gyp ERR! stack Error: Failed to execute\r\n'D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\node.exe\r\nD:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\bin\node-gyp.js\r\nconfigure --fallback-to-build\r\n--module=D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\lib\binding\Release\node-v46-win32-ia32\fse.node\r\n--module_name=fse\r\n--module_path=D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\lib\binding\Release\node-v46-win32-ia32'\r\n(1)\r\nnode-pre-gyp ERR! stack     at ChildProcess.<anonymous>\r\n(D:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\node_modules\node-pre-gyp\lib\util\compile.js:83:29)\r\nnode-pre-gyp ERR! stack     at emitTwo (events.js:87:13)\r\nnode-pre-gyp ERR! stack     at ChildProcess.emit (events.js:172:7)\r\nnode-pre-gyp ERR! stack     at maybeClose\r\n(internal/child_process.js:854:16)\r\nnode-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit\r\n(internal/child_process.js:222:5)\r\nnode-pre-gyp ERR! System Windows_NT 6.2.9200\r\nnode-pre-gyp ERR! command\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\momentjs_moment\\2.15.2\\npm\\node_modules\\.temp-v2o7tw\\node_modules\\fsevents\\node_modules\\node-pre-gyp\\bin\\node-pre-gyp"\r\n"install" "--fallback-to-build"\r\nnode-pre-gyp ERR! cwd\r\nD:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\node_modules\fsevents\r\nnode-pre-gyp ERR! node -v v4.6.2\r\nnode-pre-gyp ERR! node-pre-gyp -v v0.6.29\r\nnode-pre-gyp ERR! not ok \r\n\r\nnpm ERR! Windows_NT 6.2.9200\r\nnpm ERR! argv\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"\r\n"rebuild" "--update-binary"\r\nnpm ERR! node v4.6.2\r\nnpm ERR! npm  v3.10.9\r\nnpm ERR! code ELIFECYCLE\r\nnpm ERR! fsevents@1.0.14 install: `node-pre-gyp install\r\n--fallback-to-build`\r\nnpm ERR! Exit status 1\r\nnpm ERR! \r\nnpm ERR! Failed at the fsevents@1.0.14 install script 'node-pre-gyp\r\ninstall --fallback-to-build'.\r\nnpm ERR! Make sure you have the latest version of node.js and npm\r\ninstalled.\r\nnpm ERR! If you do, this is most likely a problem with the fsevents\r\npackage,\r\nnpm ERR! not with npm itself.\r\nnpm ERR! Tell the author that this fails on your system:\r\nnpm ERR!     node-pre-gyp install --fallback-to-build\r\nnpm ERR! You can get information on how to open an issue for this project\r\nwith:\r\nnpm ERR!     npm bugs fsevents\r\nnpm ERR! Or if that isn't available, you can get their info via:\r\nnpm ERR!     npm owner ls fsevents\r\nnpm ERR! There is likely additional logging output above.\r\n\r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!\r\nD:\home\meteor-azure\.meteor\packages\momentjs_moment\2.15.2\npm\node_modules\.temp-v2o7tw\npm-debug.log\r\n\r\n\r\nWhile building package pnyx:soap:\r\nerror: couldn't install npm package xml4js@0.1.3: Command failed:\r\nD:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\npm.cmd\r\ninstall xml4js@0.1.3\r\nnpm ERR! addLocal Could not install\r\nD:\local\Temp\npm-11544-dd92a154\git-cache-321e68d0\bb7889f02a774f347b184f3f97e8b7d8cde15e7a\r\nnpm ERR! Windows_NT 6.2.9200\r\nnpm ERR! argv\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"\r\n"install" "xml4js@0.1.3"\r\nnpm ERR! node v4.6.2\r\nnpm ERR! npm  v3.10.9\r\nnpm ERR! path\r\nD:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz\r\nnpm ERR! code ENOTSUP\r\nnpm ERR! errno -4049\r\nnpm ERR! syscall scandir\r\n\r\nnpm ERR! nospc ENOTSUP: operation not supported on socket, scandir\r\n'D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz'\r\nnpm ERR! nospc This is most likely not a problem with npm itself\r\nnpm ERR! nospc and is related to insufficient space on your system.\r\n\r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!\r\nD:\home\site\repository\packages\pnyx-soap\.npm\package-new-oh33ut\npm-debug.log\r\nnpm ERR! addLocal Could not install\r\nD:\local\Temp\npm-11544-dd92a154\git-cache-321e68d0\bb7889f02a774f347b184f3f97e8b7d8cde15e7a\r\nnpm ERR! Windows_NT 6.2.9200\r\nnpm ERR! argv\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"\r\n"D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"\r\n"install" "xml4js@0.1.3"\r\nnpm ERR! node v4.6.2\r\nnpm ERR! npm  v3.10.9\r\nnpm ERR! path\r\nD:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz\r\nnpm ERR! code ENOTSUP\r\nnpm ERR! errno -4049\r\nnpm ERR! syscall scandir\r\n\r\nnpm ERR! nospc ENOTSUP: operation not supported on socket, scandir\r\n'D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz'\r\nnpm ERR! nospc This is most likely not a problem with npm itself\r\nnpm ERR! nospc and is related to insufficient space on your system.\r\n\r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!\r\nD:\home\site\repository\packages\pnyx-soap\.npm\package-new-oh33ut\npm-debug.log\r\nD:\Program Files (x86)\SiteExtensions\Kudu\59.60207.2684\bin\Scripts\starter.cmd bash .config/azure/deploy.sh

```

**Comment by ramijarrar** [comment]
You don't need to add ```windows-build-tools```, the real issue here is with momentjs@2.15.2 which is known to be a bad release (see moment/moment#3531 & meteor/meteor#7958)

Try upgrading to momentjs@2.16.0 then rebuilding the app and let me know if the issue persists.

**Comment by one-pereyra** [comment]
It works! Thanks!! I was looking in the wrong place! 

Now I have pb with install xml4js@0.1.3. I checked it and is on the latest version  ;

> 2017-02-13T08:40:07  Updating branch 'fcebf151cdb3ca4d96f1506eaffbe3ed20489d87'.
> 2017-02-13T08:40:27  Updating submodules.
> 2017-02-13T08:42:36  Preparing deployment for commit id 'fcebf151cd'.
> 2017-02-13T08:42:37  Running custom deployment command...
> 2017-02-13T08:42:37  Running deployment command...
> 2017-02-13T08:42:37    Command: bash .config/azure/deploy.sh
> 2017-02-13T08:42:37    meteor-azure: Setting Node version
> 2017-02-13T08:42:38    Version 4.6.2 is already installed.
> 2017-02-13T08:42:39    meteor-azure: Now using Node v4.6.2 (32-bit)
> 2017-02-13T08:42:39    meteor-azure: Setting NPM version
> 2017-02-13T08:42:47    meteor-azure: Now using NPM v3.10.9
> 2017-02-13T08:42:51    meteor-azure: Installing NPM dependencies
> 2017-02-13T08:43:10    meteor-azure: Building app %DEPLOYMENT_TEMP%
> 2017-02-13T08:43:10    meteor-azure: Building app + %DEPLOYMENT_TEMP%
> 2017-02-13T08:44:34    pnyx:soap: updating npm dependencies -- xml4js, soap-vfork, jstoxml,
> 2017-02-13T08:44:34    xml2js...
> 2017-02-13T08:45:37    => Errors while initializing project:
> 2017-02-13T08:45:37    
> 2017-02-13T08:45:37    While building package pnyx:soap:
> 2017-02-13T08:45:38    error: couldn't install npm package xml4js@0.1.3: Command failed:
> 2017-02-13T08:45:38    D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\bin\npm.cmd
> 2017-02-13T08:45:38    install xml4js@0.1.3
> 2017-02-13T08:45:38    meteor-azure: ERROR! Could not generate Meteor bundle
> 2017-02-13T08:45:38    npm ERR! addLocal Could not install
> 2017-02-13T08:45:38    D:\local\Temp\npm-8088-9912c3d3\git-cache-cf9287b6\bb7889f02a774f347b184f3f97e8b7d8cde15e7a
> 2017-02-13T08:45:38    npm ERR! Windows_NT 6.2.9200
> 2017-02-13T08:45:38    npm ERR! argv
> 2017-02-13T08:45:38    "D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
> 2017-02-13T08:45:38    "D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"
> 2017-02-13T08:45:38    "install" "xml4js@0.1.3"
> 2017-02-13T08:45:38    npm ERR! node v4.6.2
> 2017-02-13T08:45:38    npm ERR! npm  v3.10.9
> 2017-02-13T08:45:38    npm ERR! path
> 2017-02-13T08:45:38    D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz
> 2017-02-13T08:45:38    npm ERR! code ENOTSUP
> 2017-02-13T08:45:38    npm ERR! errno -4049
> 2017-02-13T08:45:38    npm ERR! syscall scandir
> 2017-02-13T08:45:38    
> 2017-02-13T08:45:38    npm ERR! nospc ENOTSUP: operation not supported on socket, scandir
> 2017-02-13T08:45:38    'D:\home\meteor-azure\.meteor\packages\meteor-tool\1.4.2_3\mt-os.windows.x86_32\dev_bundle\.npm\xml2js\0.4.16\package.tgz'
> 2017-02-13T08:45:38    npm ERR! nospc This is most likely not a problem with npm itself
> 2017-02-13T08:45:38    npm ERR! nospc and is related to insufficient space on your system.
> 2017-02-13T08:45:38    
> 2017-02-13T08:45:38    npm ERR! Please include the following file with any support request:
> 2017-02-13T08:45:38    npm ERR!
> 2017-02-13T08:45:38    D:\home\site\repository\packages\pnyx-soap\.npm\package-new-1qv1wja\npm-debug.log
> 2017-02-13T08:45:38    npm ERR! addLocal Could not install
> 2017-02-13T08:45:38    D:\local\Temp\npm-8088-9912c3d3\git-cache-cf9287b6\bb7889f02a774f347b184f3f97e8b7d8cde15e7a
> 2017-02-13T08:45:38    npm ERR! Windows_NT 6.2.9200
> 2017-02-13T08:45:38    npm ERR! argv
> 2017-02-13T08:45:38    "D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe"
> 2017-02-13T08:45:38    "D:\\home\\meteor-azure\\.meteor\\packages\\meteor-tool\\1.4.2_3\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"
>

**Comment by ramijarrar** [comment]
Looks potentially similar to https://github.com/talos-code/meteor-azure/issues/3

I'm going to see if I can make a simple reproduction, then get back to you.

**Comment by one-pereyra** [comment]
that would be fantastic! see below the project's dependencies as reported with;

> for p in `meteor list | grep '^[a-z]' | awk '{ print $1"@"$2 }'`; do echo "$p"; meteor show "$p" | grep -E '^  [a-z]'; echo; done

```
accounts-password@1.1.4*

aldeed:autoform@4.2.2*

aldeed:collection2@2.8.0*

aldeed:simple-schema@1.5.3
  check@1.0.0
  deps@1.0.0
  mdg:validation-error@0.2.0
  random@1.0.0
  underscore@1.0.0

anti:i18n@0.4.3
  deps@1.0.1
  ui@1.0.0
  underscore@1.0.0

aramk:rrule@2.1.0

benmgreene:jquery-sparklines@1.1.0
  coffeescript@1.0.3
  jquery@1.0.0
  spacebars@1.0.2
  templating@1.0.7
  ui@1.0.3

browser-policy@1.0.5*

cfs:gridfs@0.0.33
  cfs:base-package@0.0.28
  cfs:storage-adapter@0.2.1

chrismbeckett:fontawesome4@4.2.2

cristo:subscription-organizer@1.0.0*

email@1.0.8*

force-ssl@1.0.6*

harrison:papa-parse@1.1.1
  jquery@1.0.0

http@1.1.1*

ian:accounts-ui-bootstrap-3@1.2.84*

iron:router@0.9.4*

jparker:crypto-md5@0.1.1
  jparker:crypto-core@0.1.0

jquery@1.11.4*

less@2.5.1*

lookback:tooltips@0.5.2
  coffeescript@1.0.6
  jquery@1.11.3_2
  reactive-var@1.0.5
  templating@1.1.0
  tracker@1.0.6

matb33:bootstrap-glyphicons@10.0.0
  isobuild:isopack-2@1.0.0
  less@2.5.0_3
  matb33:bootstrap-base@10.0.0

matb33:collection-hooks@0.8.1
  accounts-base@1.2.0 (weak dependency)
  ejson@1.0.6
  minimongo@1.0.8
  mongo@1.1.0
  tracker@1.0.7
  underscore@1.0.3

meteorhacks:kadira@2.28.5*

meteorhacks:npm@1.5.0
  meteorhacks:async@1.0.0
  underscore@1.0.2

meteorhacks:ssr@2.2.0
  blaze@2.1.3
  mongo@1.1.1
  mquandalle:jade-compiler@0.4.4 (weak dependency)
  random@1.0.4
  spacebars@1.0.7
  spacebars-compiler@1.0.7

meteorhacks:unblock@1.1.0
  meteorhacks:meteorx@1.0.2

mizzao:user-status@0.6.6
  accounts-base@1.2.1
  check@1.0.6
  coffeescript@1.0.9
  deps@1.0.9
  isobuild:isopack-2@1.0.0
  jquery@1.11.4
  mizzao:timesync@0.3.4
  mongo@1.1.1
  underscore@1.0.4

mrt:moment@2.8.1

mrt:moment-timezone@0.2.1
  mrt:moment@2.8.1

npm-container@1.2.0+

okgrow:analytics@1.0.4

percolate:synced-cron@1.3.0*

pfafman:filesaver@0.2.2*

pierreeric:rxfavico@0.3.5_1
  coffeescript@1.0.4
  jquery@1.0.1
  reactive-dict@1.0.4
  tracker@1.0.3

pnyx:boot@0.0.1+

pnyx:soap@0.0.1+

raix:handlebar-helpers@0.2.5
  deps@1.0.5
  session@1.0.4
  ui@1.0.4
  underscore@1.0.1

random@1.0.5*

reactive-dict@1.1.3*

reactive-var@1.0.6*

ros:publish-counts@0.5.1
  blaze@2.0.0 (weak dependency)
  mongo@1.0.5
  templating@1.0.6 (weak dependency)

sacha:spin@2.3.1
  templating@1.0.4 || 1.0.9
  underscore@1.0.0 || 1.0.1

standard-app-packages@1.0.6*

summernote:standalone@0.7.3*

tmeasday:html5-history-api@4.1.2

tsega:bootstrap3-datetimepicker@4.17.37_1
  jquery@1.0.0
  momentjs:moment@2.8.4

underscore@1.0.4*

universe:collection@2.2.3*

universe:ecmascript@0.6.7_1*

universe:fixes-for-third-party-packages@0.0.1
  aldeed:collection2@2.5.0 (weak dependency)
  matb33:collection-hooks@0.8.1 (weak dependency)

universe:utilities@2.3.2
  ecmascript@0.1.3
  isobuild:isopack-2@1.0.0
  minimongo@1.0.9
  tracker@1.0.8
  underscore@1.0.4

vazco:access@1.1.3+

vazco:admin@2.7.2+

vazco:autoform@1.0.1+

vazco:bootstrap-less-pnyx@0.0.0+

vazco:files-upload@0.1.2+

vazco:hopscotch@0.2.4+

vazco:maps@0.0.2

vazco:pnyx-tree-structure@0.0.0+

vazco:universe-autoform-scheduler@0.1.1+

vazco:universe-autoform-select@0.1.15+

vazco:universe-calendar@0.0.3+

vazco:universe-comments@0.0.1+

vazco:universe-core-plugin@0.0.4+

vazco:universe-html-purifier@1.2.3+

vazco:universe-mail@0.0.1+

vazco:universe-profile@1.1.5_2+

vazco:universe-reactive-queries@1.0.0+

vazco:universe-river@0.0.7+

vazco:universe-selectize@0.0.14*

vazco:universe-ui@0.6.8+

vazco:xml-template@1.0.0+

```

**Comment by ramijarrar** [comment]
We've just released v2 of the project which takes a completely different approach that addresses many of these sorts of issues people have faced with server-side bundling (as well as other significant improvements to UX and build speeds). See full announcement [here](https://forums.meteor.com/t/meteor-azure-2-0-supports-local-bundling-brand-new-docs/37266).

Please try again and open a new issue for any further problems/questions. Closing this.
